lib/ProductOpener/Food.pm

```diff
@@ -1878,7 +1878,9 @@ sub compute_nutrition_score($) {
 
 		# some categories of products do not have fibers > 0.7g (e.g. sodas)
 		# for others, display a warning when the value is missing
+		# do not display a warning if fibers are not specified on the product ('-' modifier)
 		if ((not defined $product_ref->{nutriments}{"fiber" . $prepared . "_100g"})
+			and (not defined $product_ref->{nutriments}{"fiber" . $prepared . "_modifier"})
```

Comment:
Tu n'as pas écrit le "and not" de la même manière qu'en dessous :stuck_out_tongue_closed_eyes:

Reply:
There is more than one way to do it. ;)